src/Resources/views/default/embedded_list.html.twig

```diff
@@ -9,14 +9,14 @@
 {% set widget_identifier = app.request.requestUri|embedded_list_identifier %}
 
 {% set _content_title %}
-{% spaceless %}
+{% apply spaceless %}
```

Comment:
I did not see you use `apply` tag that appeared with Twig 2.9 ... So this breaks the bundle compatibility with Twig 1.x. As it does not solve any bug only remove a deprecation, I ha ve to reject this PR.